src/i18n/strings/en_EN.json

```diff
@@ -2978,15 +2977,6 @@
     "Community %(groupId)s not found": "Community %(groupId)s not found",
     "This homeserver does not support communities": "This homeserver does not support communities",
     "Failed to load %(groupId)s": "Failed to load %(groupId)s",
-    "Great, that'll help people know it's you": "Great, that'll help people know it's you",
-    "Add a photo so people know it's you.": "Add a photo so people know it's you.",
-    "Welcome %(name)s": "Welcome %(name)s",
-    "Now, let's help you get started": "Now, let's help you get started",
-    "Welcome to %(appName)s": "Welcome to %(appName)s",
-    "Own your conversations.": "Own your conversations.",
-    "Send a Direct Message": "Send a Direct Message",
-    "Explore Public Rooms": "Explore Public Rooms",
-    "Create a Group Chat": "Create a Group Chat",
```

Comment:
why did `yarn i18n` remove these?

Reply:
Because your version of the script is too old 
You'll likely need to cull your yarn cache and reinstall on all layers
https://github.com/matrix-org/matrix-web-i18n/

Reply:
This took me a while sorry. I did something wrong with clearing the cache the first time. So I needed to yarn install on each layer twice...